tests/test_chargebee_pagination.py

```diff
@@ -17,7 +17,7 @@ def name():
 
     def generate_events(self):
         # Generate events for product catalog v1
-        url = 'https://{}.chargebee.com/api/v2/customers/cbdemo_dave'.format(os.getenv("TAP_CHARGEBEE_SITE"))
+        url = 'https://{}.chargebee.com/api/v1/customers/cbdemo_dave'.format(os.getenv("TAP_CHARGEBEE_SITE"))
```

Comment:
Is there a way to create data using `v2`? Considering that in the near future, if Chargebee plans to phase out `v1`, we'll eventually need to upgrade to `v2`.

Reply:
No, earlier tests were failing since we were using V2 endpoint and. if Chargebee phase out we will have to remove all V1 test cases.